apps/site/src/app/(main)/apply/sections/Form/Form.tsx

```diff
@@ -5,19 +5,17 @@ import axios from "axios";
 
 import Button from "@/lib/components/Button/Button";
 
-import BasicInformation from "./BasicInformation";
-import AgeInformation from "./AgeInformation";
-import SchoolInformation from "./SchoolInformation";
-import ProfileInformation from "./ProfileInformation";
-import ResumeInformation from "./ResumeInformation";
-
 import styles from "./Form.module.scss";
 import hasDeadlinePassed from "@/lib/utils/hasDeadlinePassed";
 
 const APPLY_PATH = "/api/user/apply";
 const FIELDS_WITH_OTHER = ["pronouns", "ethnicity", "school", "major"];
 
-export default function Form() {
+interface FormProps {
+	children: React.ReactNode;
+}
+
+export default function Form({ children }: FormProps) {
```

Comment:
Suggestion: you can use React's `PropsWithChildren` to specify this without defining a new interface, e.g.
function Form({ children}: PropsWithChildren).